app/src/main/java/rick_and_morty/eventbus/BusEvent.kt

```diff
@@ -3,3 +3,5 @@ package rick_and_morty.eventbus
 sealed interface BusEvent
 
 data class NavigateToCharacterDetails(val characterID: Int) : BusEvent
+object NavigateToEpisodes : BusEvent
+object NavigateToCharacters : BusEvent
```

Comment:
NIT: I think these should be ending on `...Event` (e.g.: `NavigateToCharacterDetailsEvent`) since they are extending `BusEvent` and will be posted in the `EventBus`.